addons/hr_timesheet/models/res_config_settings.py

```diff
@@ -9,8 +9,10 @@ class ResConfigSettings(models.TransientModel):
 
     module_project_timesheet_synchro = fields.Boolean("Awesome Timesheet",
         compute="_compute_timesheet_modules", store=True, readonly=False)
-    module_project_timesheet_holidays = fields.Boolean("Record Time Off",
+    module_project_timesheet_holidays = fields.Boolean("Time Off",
         compute="_compute_timesheet_modules", store=True, readonly=False)
+    reminder_user_allow = fields.Boolean(string="Employee Reminder", help="If checked, send an email to all users who have not recorded their timesheet")
+    reminder_manager_allow = fields.Boolean(string="Manager Reminder", help="If checked, send an email to all manager")
```

Comment:
What's the point of these fields?

Reply:
Currently for 'enterprise tag' we used the 'upgrade_boolen' widget and that is applied on boolean field, that's why we created these fields in hr_timesheet.

Reply:
same for 'invoice_policy'

Reply:
But these fields won't trigger any module installation. How do users know how to activate these features?

Reply:
I agree with @KangOl, no module will be installed when we enable these features. I think you need to install timesheet_grid to correctly enable the features.
But I don't know how to have a definition to install a module and another to add a group (see the definition of fields in timesheet_grid).

Reply:
Oh I think I understood. With the widget we can upgrade to have enterprise. When we have enterprise, the system will automatically install `timesheet_grid` since this module is in auto-install when `web_grid` and `hr_timesheet` and so, the features will enable as expected.  ^^
cc @KangOl and @IT-Ideas